docs/content/installation.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Installing the Ingress Controller
+
+description: 
+weight: 100
+doctypes: ["concept"]
+toc: true
+---
+
+
+This doc is now available at https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/
```

Comment:
and again -- delete and add an alias to the doc in the new location

Reply:
When I add the alias in docs/content/installation/installation-with-manifests.md to the old file docs/content/installation.md, hugo just displays the contents of the installation directory instead of redirecting the url to the new file, docs/content/installation/installation-with-manifests.md.  Do you want to rename the installation directory or just leave that alias out and let people be directed to the whole directory?

Reply:
hmm, that's not the expected behavior. I'll see if I can troubleshoot it